website/content/userguide/install/tailscale.md

```diff
@@ -53,25 +55,47 @@ have one yet), include the Tailscale daemon and CLI Go packages:
 gokr-packer \
   -update=yes \
   github.com/gokrazy/hello \
-  github.com/gokrazy/breakglass \
-  github.com/gokrazy/serial-busybox \
+  github.com/gokrazy/mkfs \
   tailscale.com/cmd/tailscaled \
   tailscale.com/cmd/tailscale
 ```
 
-## Step 3. authenticate
+We include mkfs to create filesystem on the /perm partition. mkfs only needs
+to be done once.
+
+## Step 3a. authenticate interactively
```

Comment:
Instead of naming these Step 3a and Step 3b (which could be misinterpreted as having to do both), can we make it more clear that one of the two options is enough? Maybe “Step 3 Alternative A” and “Step 3 Alternative B”?

Reply:
I updated based on bradfitz comment re: tailscale v1.22.1 and also edited the steps.

Hopefully this version is better.